gammapy/cube/tests/test_psf_map.py

```diff
@@ -106,3 +160,27 @@ def test_containment_radius_map(tmpdir):
     val = m.interp_by_coord(coord)
 
     assert_allclose(val, 0.227463, rtol=1e-3)
+
+
+def test_psfmap_stacking():
+    psfmap1 = build_test_psfmap(0.1 * u.deg, shape="flat")
+    psfmap2 = build_test_psfmap(0.1 * u.deg, shape="flat")
+    psfmap2.exposure_map.quantity *= 2
+
+    psfmap_stack = psfmap1.stack(psfmap2)
+    assert_allclose(psfmap_stack.data, psfmap1.data)
+    assert_allclose(psfmap_stack.exposure_map.data, psfmap1.exposure_map.data * 3)
+
+    psfmap3 = build_test_psfmap(0.3 * u.deg, shape="flat")
+    psfmap_stack = psfmap1.stack(psfmap3)
+
+    assert_allclose(psfmap_stack.data[:, 40, :, :], 0.0)
+    assert_allclose(
+        psfmap_stack.data[:, 20, :, :], psfmap3.data[:, 20, :, :] * 0.5, rtol=1e-4
```

Comment:
As you know, I think explicit / shorter asserts on the exact value of the result are better.
Since all pixel values are the same, you can just pick one pixel here (and then avoid the line break).
Also setting reduced `rtol=1e-4` shouldn't be necessary, that's a simple multiply of to float values, the outtput should be super-precise, no?

Reply:
OK done. Only test at center to avoid regions where value is 0. And use explicit expected value.